decidim-admin/app/assets/stylesheets/decidim/admin/modules/_secondary-nav.scss

```diff
@@ -51,7 +51,8 @@ $process-title-height: 3rem;
 }
 
 .secondary-nav__title,
-.process-title-content{
+.process-title-content,
+.scope-title-content{
```

Comment:
For the record, now they live on:
https://github.com/decidim/design
https://github.com/decidim/design-admin